browsers/node16.5.0-chrome94-ff93/Dockerfile

```diff
@@ -50,4 +50,4 @@ ENV TERM xterm
 # avoid million NPM install messages
 ENV npm_config_loglevel warn
 # allow installing when the main user is root
-ENV npm_config_unsafe_perm true
+ENV npm_config_unsafe_perm true
```

Comment:
Not a huge deal but looks like this file doesn't need to be touched. Looks like a prettier formatting thing